android-interop-testing/proguard-rules.pro

```diff
@@ -21,3 +21,7 @@
 
 # Ignores: can't find referenced method from grpc-testing's compileOnly dependency on Truth
 -dontwarn io.grpc.testing.DeadlineSubject
+
+# Ignores: can't find referenced method for java.nio.ByteBuffer limit(int) due to JDK ABI
+# incompatibility.
+-dontwarn java.nio.ByteBuffer
```

Comment:
This will not be needed once #6839 is in.

Reply:
Yep

Reply:
Reverted.